docs/sampling/README.md

```diff
@@ -0,0 +1,242 @@
+Threshold and Randomness properties are called "T-value" and
+"R-value"; they are represented in the OpenTelemetry section of the
+`tracestate` (having vendor tag `ot`), using properties named `th` and
+`rv`, respectively.
+
+For the logs signal, which generally does not record the W3C Trace
+Context `tracestate`, sampling attributes are meant to be expressed
+using log record attributes with the same definition as T-value and
+R-value.
+
+For more information on how to perform and interpret probability
+sampling based on these properties, [consult OTEP 235][OTEP235].
+
+### Overriding sampling decisions
+
+Instrumentation authors and end-users that wish to prioritize an item
+of telemetry for collection in spite of sampling can add a
+`sampling.priority` attribute.  This attribute is inherited as a
+legacy from the OpenTracing project, which gave the following
+definition.
+
+> If greater than 0, a hint to the Tracer to do its best to capture
+> the trace. If 0, a hint to the Tracer to not-capture the trace. If
+> absent, the Tracer should use its default sampling mechanism.
+
+OpenTelemetry users can think of this attribute as a suggestion, a way
+of recognizing the importance of a certain event and requesting
+additional consideration from the collection pipeline.  For example, a
+user could write this code snippet:
+
+```
+   if err := doSomething(); err != nil {
+     if err == VERY_SERIOUS {
+       span.SetAttribute("sampling.priority", 10000)
+     }
+     return err
+   }
+```
+
+Samplers and sampling processors SHOULD pass items of telemetry to the
+exporter, independent of their default sampling mechanism, when the
+`sampling.priority` attribute is present and non-zero.  Although the
+priority is an integer, mathematical weight is not prescribed and no
+other specific behaviors are required.
+
+### Overriding sampling randomness
+
+Sampling system designers are able to override sampling randomness on
+an item-by-item basis, which may be done for several reasons, including
+situations where there is no TraceID defined.
+
+When a tracing system purposely uses TraceIDs that do not follow the
+W3C Trace Context Level 2 specification for TraceID randomness, and
+they wish to use OpenTelemetry sampling components, they can insert
+explicit randomness to prevent erroneously taking randomness from the
+TraceID.
+
+Another use for overriding sampling randomness is to configure a
+different unit of sampling consistency.  For example, multiple traces
+can be given the same randomness value to ensure that either all or
+none of them are sampled consistently.
+
+In the tracing signal, sampling randomness can be overridden by
+setting an R-value in the tracestate.  In the logging signal, sampling
+randomness can be overridden by setting the `sampling.randomness`
+attribute.
+
+### Sampling threshold
+
+When determining the Threshold value from an item of telemetry,
+implementations SHOULD:
```

Comment:
I still don't understand implementations of what. Sampler? Processor? Exporter? Collector/etc pipeline component?

Reply:
In the prototype for probabilistictailsampler, https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/24811, the processor is an "intermediate span sampler" which may both read the threshold and write a new threshold after making a new sampling decision. So, we expect this to be used by head samplers, tail samplers, and backends for constructing metrics from sampled items. I also wonder if the changes in https://github.com/open-telemetry/opentelemetry-specification/pull/3910 help answer this sort of question?

Reply:
I added a tail-sampler example here: [4e8870d](https://github.com/open-telemetry/semantic-conventions/pull/793/commits/4e8870d74da0cc35f80737f7800a096a4000c15d)